client/src/pages/signin.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Button, TextField, Paper, FormControl } from '@material-ui/core';
+import laptopPhoto from '../images/blue-shirt-at-laptop.png'
+import { Link } from 'react-router-dom';
+import './signin.css';
+
+export default class Signin extends React.Component {
+    render() {
+        return (
+            <div className="main">
```

Comment:
When using Material UI, try to use as many of its components as possible instead of generic HTML components.
`div`s can be [Grid](https://material-ui.com/components/grid/) and try to put all text in [Typography](https://material-ui.com/components/typography/). Whenever making a new component, just take a look at the list of components in Material UI and see if there's something you can use. Consistently using their components means a lot of styling will be built into the components already and it will make life easier